components/script/dom/imagebitmap.rs

```diff
@@ -0,0 +1,76 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+// the following are the statements
```

Comment:
This comment does not add useful information. Please remove it.

Reply:
Removed in the code.